static/js/map.js

```diff
@@ -1414,68 +1452,81 @@ function setupPokestopMarker(item) {
-        imagename = 'PokestopLuredMagnetic'
-    } else if (lure) {
-        imagename = 'PokestopLured'
-    }
-
-	var icon =  L.icon({
-        iconUrl: 'static/images/pokestop/' + imagename + '.png',
+function updatePokestopMarker(pokestop, marker) {
+    var questRewardType = pokestop['quest_raw']['quest_reward_type_raw']
+    var questItemId = pokestop['quest_raw']['item_id']
+    var questPokemonId = pokestop['quest_raw']['pokemon_id']
+    var lureExipiration = pokestop['lure_expiration']
+    var activeFortModifier = pokestop['active_fort_modifier']
+    var markerImage
+    var shadowImage
+    var shadowSize
+    var shadowAnchor
+
+    if (Store.get('showQuests') && questRewardType) {
+        switch (questRewardType) {
+            case '2':
+                if (excludedQuestItems.indexOf(parseInt(questItemId)) === -1) {
+                    shadowImage = 'static/images/quest/reward_' + questItemId + '_1.png'
+                    shadowSize = [30, 30]
+                    shadowAnchor = [30, 20]
+                    markerImage = 'pokestop_quest'
+                }
+                break
+            case '3':
+                if (excludedQuestItems.indexOf(6) === -1) {
+                    shadowImage = 'static/images/quest/reward_stardust.png'
+                    shadowSize = [30, 30]
+                    shadowAnchor = [30, 20]
+                    markerImage = 'pokestop_quest'
+                }
+                break
+            case '7':
+                if (excludedQuestPokemon.indexOf(parseInt(questPokemonId)) === -1) {
+                    if (generateImages) {
+                        shadowImage = `pkm_img?pkm=${questPokemonId}`
+                        shadowSize = [35, 35]
+                        shadowAnchor = [30, 30]
+                    } else {
+                        shadowImage = pokemonSprites(questPokemonId).filename
+                        shadowSize = [40, 40]
+                        shadowAnchor = [30, 30]
+                    }
+                    markerImage = 'pokestop_quest'
+                }
+        }
+    }
+
+    if (Store.get('showPokestops') && lureExipiration) {
+        if (Store.get('showLuredPokestopsOnly') <= 2 && !pokestop['active_fort_modifier']) {
+            markerImage = 'pokestop_lured'
+        } else if ((Store.get('showLuredPokestopsOnly') <= 1 || Store.get('showLuredPokestopsOnly') == 3) && activeFortModifier === 502) {
+            markerImage = 'pokestop_lured_glacial'
+        } else if ((Store.get('showLuredPokestopsOnly') <= 1 || Store.get('showLuredPokestopsOnly') == 4) && activeFortModifier === 504) {
+            markerImage = 'pokestop_lured_magnetic'
+        } else if ((Store.get('showLuredPokestopsOnly') <= 1 || Store.get('showLuredPokestopsOnly') == 5) && activeFortModifier === 503) {
+            markerImage = 'pokestop_lured_mossy'
+        }
+    }
+
+    if (!markerImage) {
+        markerImage = 'pokestop'
```

Comment:
I get 404 errors because the image is called `Pokestop` and not `pokestop`. Could you please rename this image too? :upside_down_face: